### DISCUSSION TITLE
Q: Is there anyway to update KadeEngine even if source codes were modified?

### DISCUSSION THREAD

**Discussion by FaultyTwo** [KadeArchive/Kade-Engine#461] [Q&A]
**For example:** 
Let's say I have a mod (with all of its source codes) based on KadeEngine version whatever. And I want to update it to the latest version of KadeEngine. Do I HAVE to do everything from ground up with newest version, or is there anyway to update it without losing any modifications?

If there're some fancy git commands that I never heard of and does exactly that. Please tell me. Thanks!

**Comment by puyoxyz** [comment]
If you forked using GitHub, there's this button at the top:  ![image](https://user-images.githubusercontent.com/15311104/118290758-13515e00-b4d7-11eb-801b-beab3b6fd3ff.png)

If not, follow [this guide](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/merging-an-upstream-repository-into-your-fork)

**Reply by FaultyTwo** [reply]
Got it, thank you.
